Moblin/View/Settings/RtmpServer/RtmpServerStreamSettingsView.swift

```diff
@@ -136,12 +133,13 @@ struct RtmpServerStreamSettingsView: View {
                 }))
                 .disabled(model.rtmpServerEnabled())
                 if stream.manualFps! {
-                    TextEditNavigationView(
-                        title: String(localized: "FPS"),
-                        value: String(stream.fps!),
-                        onSubmit: submitFps,
-                        keyboardType: .numbersAndPunctuation
-                    )
+                    Picker(String(localized: "FPS"), selection: Binding(get: {
```

Comment:
`String(localized: "FPS")` can be replaced with `"FPS"` here.

Reply:
done